### PR TITLE
docs: update chainhook guides

### PR DESCRIPTION
**Pull request by mefrem**
### Description

@mefrem: Fixed a pathname error in docs, fixed a typo, and added some clarifying language to other sections.

- Fixed `bitcoind` and `bitcoin.conf` pathnames
- Removed double words "predicates" 
- Clarified bitcoind node setup instructions

@ryanwaits to add changes/comments to the "Run Chainhook as a Service using Stacks" How To guide

### CODE REVIEW

File: docs/how-to-guides/how-to-run-chainhook-as-a-service-using-stacks.md [hunk 1]
```diff
@@ -48,31 +48,43 @@ events_keys = ["*"]
...
-stacks_node_rpc_url = "http://localhost:20443"            # Must match with the rpc_bind in the Stacks.toml file
+bitcoind_rpc_url = "http://localhost:8332"
+bitcoind_rpc_username = "devnet"
+bitcoind_rpc_password = "devnet"
+# Bitcoin block events can be received by Chainhook
+# either through a Bitcoin node's ZeroMQ interface,
+# or through the Stacks node. The Stacks node is
+# used by default:
+stacks_node_rpc_url = "http://localhost:20443"
+stacks_events_ingestion_port = 20455
```
Comment (ryanwaits) [inline docs/how-to-guides/how-to-run-chainhook-as-a-service-using-stacks.md:85]: this configuration was mentioned in the previous version of the docs, but was never present in the `Chainhook.toml`

i was also unable to reproduce any meaningful differences by including or excluding, so would love to get more clarity around the `stacks_events_ingestion_port`

Comment (mefrem) [reply docs/how-to-guides/how-to-run-chainhook-as-a-service-using-stacks.md:85]: I think this is something @MicaiahReid could comment on

**Review by MicaiahReid — CHANGES_REQUESTED** [review]
Great improvement! I have a few nits.

File: docs/how-to-guides/how-to-run-chainhook-as-a-service-using-bitcoind.md [hunk 2]
```diff
@@ -8,27 +8,26 @@ You can run Chainhook as a service to evaluate your `if_this / then_that` predic
...
 
-This guide is written to work with the latest Bitcoin Core software containing bitcoind, [Bitcoin Core 25.0](https://bitcoincore.org/bin/bitcoin-core-25.0/).
+This guide is written to work with the latest Bitcoin Core software containing bitcoind, [Bitcoin Core 25.0](https://bitcoincore.org/bin/bitcoin-core-25.0/). While bitcoind can and will start syncing a Bitcoin node, customizing this node to your use cases beyond supporting a Chainhook is out of scope for this guide. See the Bitcoin wiki for ["Running Bitcoin"](https://en.bitcoin.it/wiki/Running_Bitcoin) or bitcoin.org [Running A Full Node guide](https://bitcoin.org/en/full-node).
 
 > **_NOTE:_**
 >
-> While bitcoind can and will start syncing a Bitcoin node, customizing this node to your use cases beyond supporting a Chainhook is out of scope for this guide. See the Bitcoin wiki for ["Running Bitcoin"](https://en.bitcoin.it/wiki/Running_Bitcoin) or bitcoin.org [Running A Full Node guide](https://bitcoin.org/en/full-node).
+> The OS X .tar.gz file that includes bitcoind can be downloaded directly in terminal with the following command:
```
[the remark this answers is not in the document]
Comment (mefrem) [reply docs/how-to-guides/how-to-run-chainhook-as-a-service-using-bitcoind.md:17]: I believe I thought I was following a convention I saw elsewhere in docs. I wanted to motivate using `curl`—I'll remove the OS X reference.

File: docs/how-to-guides/how-to-run-chainhook-as-a-service-using-bitcoind.md [hunk 3]
```diff
@@ -52,18 +51,17 @@ dbcache=4096
...
-
 > **_NOTE:_**
 >
 > The below command is a startup process that, if this is your first time syncing a node, might take a few hours to a few days to run. Alternatively, if the directory pointed to in the `datadir` field above contains bitcoin blockchain data, syncing will resume.
 
+Now that you have `bitcoin.conf` file ready with the bitcoind configurations, you can run the bitcoind node. The command takes the form `<path/to/bitcoind> -conf=<path/to/bitcoin.conf>`, for example:
+
 ```console
-./bitcoind -conf=</path/to/bitcoin/directory/>
+/Volumes/SSD/bitcoin-25.0/bin/bitcoind -conf=</Volumes/SSD/project/Chainhook/bitcoin.conf>
```
[the remark this answers is not in the document]
Comment (mefrem) [reply docs/how-to-guides/how-to-run-chainhook-as-a-service-using-bitcoind.md:61]: Yep, we should

File: docs/how-to-guides/how-to-run-chainhook-as-a-service-using-stacks.md [hunk 4]
```diff
@@ -48,31 +48,43 @@ events_keys = ["*"]
...
 ```toml
 [storage]
 working_dir = "cache"
 
-# The HTTP API allows you to register/deregister
-# dynamically predicates
-# Disable by default
+# The Http Api allows you to register / deregister
+# dynamically predicates.
+# Disable by default.
```
[the remark this answers is not in the document]
Comment (mefrem) [reply docs/how-to-guides/how-to-run-chainhook-as-a-service-using-stacks.md:69]: Including this change in the docs code block, and yet a permanent fix would be at the config generator source code.

File: docs/how-to-guides/how-to-run-chainhook-as-a-service-using-stacks.md (hunk 1 above)
Comment (mefrem) [reply docs/how-to-guides/how-to-run-chainhook-as-a-service-using-stacks.md:85]: Indeed, Micaiah spoke to it below, see comment: https://github.com/hirosystems/chainhook/pull/384#discussion_r1296005638

Comment (mefrem) [reply docs/how-to-guides/how-to-run-chainhook-as-a-service-using-stacks.md:85]: @ryanwaits — is there anything you think should be added to the Chainhook.toml generator [PR](https://github.com/hirosystems/chainhook/pull/382) ([link to generator.rs file](https://github.com/hirosystems/chainhook/blob/improvement/cli-config-generator/components/chainhook-cli/src/config/generator.rs))to add clarity to this?

**Comment by mefrem** [comment]
All changes suggested by @MicaiahReid look good to me, manually added. Would love to merge and get one more Docs release update before webinar.

**Pull request merged**